src/Evolution/Systems/Burgers/BoundaryConditions/Dirichlet.hpp

```diff
@@ -46,7 +46,7 @@ class Dirichlet final : public BoundaryCondition {
   using options = tmpl::list<U>;
   static constexpr Options::String help{
       "Dirichlet boundary conditions setting the value of U to "
-      "time-independent constants."};
+      "a time-independent constants."};
```

Comment:
1. previous line: "conditions" -> "condition"
2. this line: "constants" -> "constant"